src/main/resources/mustache-templates/layout/footer.html

```diff
@@ -19,18 +19,17 @@
 	</div>
 </footer>
 
+ <script type="text/javascript"
+	src="{{req.contextPath}}/webjars/bootstrap/js/bootstrap.min.js"></script>
 <script type="text/javascript"
 	src="{{req.contextPath}}/webjars/jquery/jquery.min.js"></script>
-<script type="text/javascript"
-	src="{{req.contextPath}}/webjars/bootstrap/js/bootstrap.min.js"></script>
-
-<!--/* Set the active navigation */-->
-<script type="text/javascript">
-	$(function() {
-		var pathname = window.location.pathname;
-		$('.nav a[href$="' + pathname + '"]').parent().addClass('active');
-	});
-</script>
+{{#includeHpoSummaryJs}}
+	<script type="text/javascript"
+		src="{{req.contextPath}}/webjars/datatables/js/jquery.dataTables.min.js"></script>
+	<script type="text/javascript"
+		src="{{req.contextPath}}/webjars/datatables/js/dataTables.bootstrap4.min.js"></script>
+	<script src="/js/hposummary.js"></script>
```

Comment:
You might need to include the contextPath here as well. Seems like I ran into some issues with something like this in deployment.